components/SiteTableOfContents.vue

```diff
@@ -0,0 +1,20 @@
+<script setup>
+import { useSidebar } from 'vitepress/theme'
+
+const { sidebar } = useSidebar()
+
+function getHref(link) {
+  return `${link}.html`
+}
+</script>
+
+<template>
+  <template v-for="group of sidebar">
+    <h3>{{ group.text }}</h3>
+    <ul>
+      <li v-for="item of group.items">
+        <a :href="getHref(item.link)">{{ item.text }}</a>
+      </li>
+    </ul>
+  </template>
```

Comment:
Didn't find how to interpret Markdown so had to generate HTML. Unfortunately lose anchors but that's ok.